sources/packages/api/src/services/ssh/ssh.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable, Scope } from "@nestjs/common";
+import { SFTPConfig } from "../../types";
+import * as Client from "ssh2-sftp-client";
+
+@Injectable()
+export class SshService {
```

Comment:
Any method to close connection?

Reply:
Added the closed connection and checked the connections on the SFTP server.